.ci/on_conan_release.jenkinsfile

```diff
@@ -26,4 +26,72 @@ node('Linux') {
             }
         }
     }
+
+    stage('Upgrade default conan version') {
+        // Here we need to create a PR modifying the Conan version
+
+        // Clone, save and check if diff
+        checkout([$class           : 'GitSCM',
+                  branches         : [[name: '*/master']],
+                  userRemoteConfigs: [[credentialsId: 'conanci-gh-token', url: 'https://github.com/conan-io/conan-docker-tools.git']],
+                  extensions       : [[$class           : 'RelativeTargetDirectory',
+                                       relativeTargetDir: 'conan-docker-tools']],
+        ])
+
+        {
+            def content = readFile(file: 'conan-docker-tools/modern/.env')
+            content = content.replaceAll(/CONAN_VERSION=\d+\.\d+.\d+/, "CONAN_VERSION=${params.conan_version}")
+            content = content.replaceAll(/DOCKER_TAG=\d+\.\d+.\d+/, "DOCKER_TAG=${params.conan_version}")
+            writeFile(file: 'conan-docker-tools/modern/.env', text: content)
+        }
+
+        {
+            def content = readFile(file: 'conan-docker-tools/legacy/.env')
+            content = content.replaceAll(/CONAN_VERSION=\d+\.\d+.\d+/, "CONAN_VERSION=${params.conan_version}")
+            content = content.replaceAll(/DOCKER_BUILD_TAG=\d+\.\d+.\d+/, "DOCKER_BUILD_TAG=${params.conan_version}")
+            writeFile(file: 'conan-docker-tools/legacy/.env', text: content)
+        }
+
+        String branchName = "bot/release-${params.conan_version}".replaceAll(/\./, '_')
+        dir('conan-docker-tools') {
+            sh "git checkout -b ${branchName}"
+            sh "git diff"
+
+            sh 'git config --local user.email "javierg@jfrog.com"'
+            sh 'git config --local user.name "Release Bot"'
+            sh 'git commit -am "[up conan] Update Conan version"'
+
+            if (params.build_old_images) {
+                withCredentials([usernamePassword(credentialsId: 'conanci-gh-token', usernameVariable: 'GH_USER', passwordVariable: 'GH_PASS')]) {
+                        sh "git remote set-url origin https://$GH_USER:$GH_PASS@github.com/conan-io/conan-docker-tools.git"
+                        sh "git push origin ${branchName}"
+                }
+            }
+        }
+
+        // Open PR in Github
+        stage('Open PR in Github') {
+            def data = readJSON text: '{}'
+            data.title = "[up conan] Update Conan version (${params.conan_version})" as String
+            data.head = branchName
+            data.base = 'master' as String
+            data.body = '''\
+                New Conan version released.
+
+                Please, @conan-io/barbarians have a look to this PR
```

Comment:
I think we could create a docker group. Barbarians will alert all conan team.

Reply:
We can ask for actual reviews from certain users (I'll do that)

Reply:
nice!